### PR TITLE
Modularize package.py

### PR DESCRIPTION
**Pull request by dirkmueller**
This has multiple thousand lines of code. we can split each container in its own package. this starts with the first two.

### CODE REVIEW

File: src/bci_build/package/basalt_base.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+"""Base Container for the Basalt Project"""
+from __future__ import annotations
```
Comment (dcermak) [inline src/bci_build/package/basalt_base.py:2]: This shouldn't be necessary
```suggestion
```

File: src/bci_build/package/python.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
```
Comment (dcermak) [inline src/bci_build/package/python.py:3]:
```suggestion
```

**Review by dcermak — DISMISSED** [review]
LGTM

File: src/bci_build/package/basalt_base.py (hunk 1 above)
Comment (dirkmueller) [reply src/bci_build/package/basalt_base.py]: included in the rebase, thanks!

File: src/bci_build/package/python.py (hunk 2 above)
Comment (dirkmueller) [reply src/bci_build/package/python.py]: included in the rebase.

**Pull request merged**